wp-content/themes/access/functions.php

```diff
@@ -69,3 +69,4 @@
 // phpcs:enable
 
 require_once ACCESS\lib('filters');
+require_once ACCESS\lib('feedback');
```

Comment:
I would prefer if **lib/feedback.php** was an mu-plugin; **wp-content/mu-plugins/feedback.php**

If you get a chance to move it there, you won't need this require statement.